features/graph/api/src/main/java/org/opennms/netmgt/graph/api/Vertex.java

```diff
@@ -30,7 +30,15 @@
 
 import org.opennms.netmgt.graph.api.generic.GenericVertex;
 
-public interface Vertex extends VertexRef {
+public interface Vertex {
 
     GenericVertex asGenericVertex();
+
+    VertexRef getVertexRef();
+
+    // TODO: patrick discuss with mvr: shall we keep is method as a convenient way to say getVertexRef.getNamespace()
+    String getNamespace();
```

Comment:
@mvrueden to be discussed

Reply:
Let's keep it for now. 
We can - if all features are implemented - still rework bits and pieces.